### ISSUE TITLE
Meta-analysis feature.

### ISSUE THREAD

**Issue by chrisgorgo** [NeuroVault/NeuroVault#627]
Add an image-based meta-analysis feature to NeuroVault, allowing analysis of selected group maps
or single subject maps from different studies (pooled analysis). Should be available for private collections.

Question - how can users access private collections? A meta-collection? Consortium?

**Comment by tyarkoni** [comment]
I think this is within the scope of the Neurosynth renewal... You're welcome to work on it before then if you like, but my thinking was that we would build an entirely new interface for Neurosynth that ingests both coordinate- and image-based data from NeuroVault, Brainspell, and potentially other sources. That way there would be a modular structure: NeuroVault for data, Brainspell for annotation, and Neurosynth for meta-analysis, with all of these things talking to each other nicely.

One problem we will need to solve for all of this to work nicely is develop a common standard for representing study information. This has already come up in neurostuff/NiMARE#13, and I suggested to @tsalo that we draft a BIDS extension to handle representation of study information (including coordinates, images, annotations, etc.). This is probably something that should have a lot of input from folks though, so I'm not sure what the best way to proceed is.

**Comment by chrisfilo** [comment]
Yup - this sounds like a good plan of achieving this. I added the issue to track functionality being part of the NutBrain subcontract.
